node/src/sub_lib/accountant.rs

```diff
@@ -94,6 +94,8 @@ pub struct AccountantSubs {
     pub report_routing_service_provided: Recipient<ReportRoutingServiceProvidedMessage>,
     pub report_exit_service_provided: Recipient<ReportExitServiceProvidedMessage>,
     pub report_services_consumed: Recipient<ReportServicesConsumedMessage>,
+    pub report_our_balances_and_qualified_payables:
```

Comment:
It would be better to rename it to `report_balance_and_qualified_payables`

Reply:
I will change this one...

But accordingly to the final decision about the greater naming.

Reply:
Ok, let's do that.